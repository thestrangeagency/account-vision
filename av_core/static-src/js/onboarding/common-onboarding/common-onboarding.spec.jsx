import React from 'react';
import { shallow } from 'enzyme';

import CommonOnboarding from './common-onboarding';

test('it renders', () => {
  const wrapper = shallow(<CommonOnboarding nextPage="" returnYear="" />);
  expect(wrapper).toMatchSnapshot();
});
