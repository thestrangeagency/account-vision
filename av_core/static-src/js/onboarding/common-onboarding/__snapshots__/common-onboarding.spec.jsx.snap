// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div
  className="common-onboarding"
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <h4
        className="my-3"
      >
        Do you have any of these expenses?
      </h4>
      <CommonExpenseManager
        buttonText="Next"
        forceButtonEnabled={true}
        onSaved={[Function]}
        year=""
      />
    </div>
  </div>
</div>
`;
